test/Data/Time/Format/HumanSpec.hs

```diff
@@ -89,4 +96,4 @@ spec = describe "humanReadableTime'" $ do
         humanReadableTime' t n `shouldBe` "on Dec 30, 2025"
 
 parseTime' :: String -> UTCTime
-parseTime' = fromJust . parseTime defaultTimeLocale "%F %T%Q"
+parseTime' = fromJust . parseTimeM True defaultTimeLocale "%F %T%Q"
```

Comment:
No functional change. Just that `parseTime` was deprecated in time-1.5 in favour of `parseTimeM`.